src/App.js

```diff
@@ -1,23 +1,53 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { Route, NavLink, Switch } from "react-router-dom";
+import Menu from "./components/Menu";
+import Order from "./components/Order.js";
+import Bill from "./components/Bill.js";
+import Service from "./components/Service.js";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header className="app__header">
+        <Switch>
+          <Route path="/menu">
+            <Menu />
+          </Route>
+          <Route path="/order">
+            <Order />
+          </Route>
+          <Route path="/bill">
+            <Bill />
+          </Route>
+          <Route path="/service">
+            <Service />
+          </Route>
+        </Switch>
       </header>
+      <main className="app__main">
+        <p>Dish 1</p>
+        <p>Dish 2</p>
+        <p>Dish 3</p>
+        <p>Dish 4</p>
+        <p>Dish 5</p>
+        <p>Dish 6</p>
+      </main>
+      <footer className="app__footer">
+        <nav className="app__nav">
+          <NavLink className="app__navlink" to="/menu">
+            Menu
+          </NavLink>
+          <NavLink className="app__navlink" to="/order">
+            Order
+          </NavLink>
+          <NavLink className="app__navlink" to="/bill">
+            Bill
+          </NavLink>
+          <NavLink className="app__navlink" to="/service">
+            Service
+          </NavLink>
+        </nav>
```

Comment:
I'd also make this its own component (maybe you have it planned as  a next step)